packages/opentelemetry-resources/src/platform/node/detectors/EnvDetector.ts

```diff
@@ -43,50 +43,55 @@ class EnvDetector implements Detector {
     ' characters.';
 
   /**
-   * Returns a {@link Resource} populated with labels from the
-   * OTEL_RESOURCE_LABELS environment variable. Note this is an async function
-   * to conform to the Detector interface.
+   * Returns a {@link Resource} populated with attributes from the
+   * OTEL_RESOURCE_ATTRIBUTES environment variable. Note this is an async
+   * function to conform to the Detector interface.
    *
    * @param config The resource detection config with a required logger
    */
   async detect(config: ResourceDetectionConfigWithLogger): Promise<Resource> {
     try {
-      const labelString = process.env.OTEL_RESOURCE_LABELS;
-      if (!labelString) {
+      const rawAttributes = process.env.OTEL_RESOURCE_ATTRIBUTES;
```

Comment:
I'm wondering one thing here, above all should this be a part of `environment` instead of accessing `process.env` directly. I know this inn under platform node, but still I would be in favour of getting env variables through `getEnv()`, I'm fine to do it in next PR as it was already like this before, WDYT ?

Reply:
I agree, but I think it is outside the scope of this PR.